### PR TITLE
Add introductional paragraph, fix share names

### PR DESCRIPTION
**Pull request by ThomDietrich**
share names with spaces may cause problems

### CODE REVIEW

File: installation/rasppi.md [hunk 1]
```diff
@@ -43,18 +45,18 @@ wins support = yes
 then add the following to the share definitions section (scroll all the way down to the bottom of the long file):
 
 ```
-[openHAB Home]
- comment= openHAB Home
- path=/usr/share/openhab
+[openHAB]
+ comment=OpenHAB2 application
```
Comment (kaikreuzer) [inline installation/rasppi.md]: openHAB spelled with a small 'o'!

File: installation/rasppi.md [hunk 2]
```diff
@@ -43,18 +45,18 @@ wins support = yes
...
  writeable=Yes
  only guest=no
  create mask=0777
  directory mask=0777
  public=no
-[openHAB Config]
- comment= openHAB Site Config
- path=/etc/openhab
+ [openHAB-etc]
+ comment=OpenHAB2 site configuration
```
Comment (kaikreuzer) [inline installation/rasppi.md]: dito

**Comment by ThomDietrich** [comment]
fixed

**Comment by kaikreuzer** [comment]
many thanks!

**Pull request merged**